src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs

```diff
@@ -279,7 +279,7 @@ protected override PipelineInstruction PipelineCallback(PipelineEntry? entry, Re
             // If we are already logged in and the server returns 530 then
             // the server does not support re-issuing a USER command,
             // tear down the connection and start all over again
-            if (entry.Command.IndexOf("USER", StringComparison.Ordinal) != -1)
+            if (entry.Command.Contains("USER"))
```

Comment:
Shouldn't we keep the `StringComparison.Ordinal`?

Reply:
The analyzer suggested this. My assumption is that `Ordinal` is the default comparison when using `Contains`.

Reply:
@tarekgh is this assumption correct?

Reply:
String.Contains behavior is doing ordinal comparison by default. We don't need to pass `StringComparison.Ordinal`.